### PR TITLE
Update test-linux-gpu.yml

### PR DESCRIPTION
**Pull request by vfdev-5**
Fixes CUDA drivers issue with test-linux-gpu job

### CODE REVIEW

**Comment by vfdev-5** [comment]
@osalpekar do you know why the following can happen : https://github.com/pytorch/vision/actions/runs/3648497295/attempts/1

File: .github/workflows/test-linux-gpu.yml [hunk 1]
```diff
@@ -51,8 +51,11 @@ jobs:
         conda install \
           --yes \
           -c "pytorch-${CHANNEL}" \
-          -c nvidia "pytorch-${CHANNEL}"::pytorch[build="*${VERSION}*"] \
+          -c nvidia pytorch \
```
Comment (pmeier) [inline .github/workflows/test-linux-gpu.yml:54]: Let's separate the channels from the packages we want to install

```suggestion
          -c nvidia \
          pytorch \
```

I'm also ok with having both channels and packages on one line together

**Comment by pmeier** [comment]
There are some OOM errors for the model tests: https://github.com/pytorch/vision/actions/runs/3648584911/jobs/6162177024#step:9:35363

**Comment by pmeier** [comment]
It seems on `linux.4xlarge.nvidia.gpu` we are getting OOM errors, while on `linux.g5.4xlarge.nvidia.gpu` the driver is not set up properly.

**Comment by osalpekar** [comment]
> @osalpekar do you know why the following error can happen : https://github.com/pytorch/vision/actions/runs/3648497295/attempts/1 ? image

cc @seemethere or @atalman Looks like cleaning the workspace in the generic job fails due to permissions issues with certain directories. @lequytra also saw this while migrating CI jobs in torchrec: https://github.com/pytorch/torchrec/actions/runs/3651587724/jobs/6168971846.

**Comment by vfdev-5** [comment]
Outdated PR

**Pull request closed without merging**